fastlane/spec/tools_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Fastlane do
+  describe "Fastlane::TOOLS" do
+    it "lists all the fastlane tools" do
+      expect(Fastlane::TOOLS.count).to be >= 15
```

Comment:
I don't think it's useful to have a list of the tools at 2 locations. That's why I just check if it's an array and there are tools inside.